stories/components/Timesheet/TimesheetSummary.spec.tsx

```diff
@@ -26,9 +25,7 @@ const {
 } = composeStories(stories);
 
 test("Default", () => {
-  const { container } = render(
-    <Default {...(Default.args as ITimesheetSummaryProps)} />
```

Comment:
There is no need to use args from the story because it's already contained in the component returned by composeStories! It should just work!